tinygrad/tensor.py

```diff
@@ -1343,6 +1343,28 @@ def chunk(self, chunks:int, dim:int=0) -> List[Tensor]:
     dim = self._resolve_dim(dim)
     return list(self.split(ceildiv(self.shape[dim], chunks) if self.shape[dim] else [0]*chunks, dim=dim))
 
+  def meshgrid(self:Tensor, *args:Tensor, indexing:Union[Literal["ij"], Literal["xy"]]="ij") -> Tuple[Tensor, ...]:
+    """
+    Generates coordinate matrices from coordinate vectors.
+
+    When `indexing` is `"ij"` the dimensions are in the same order as the cardinality of the inputs.
+    When `indexing` is `"xy"` the cardinality of first and second dimensions are swapped.
+
+    ```python exec="true" source="above" session="tensor" result="python"
+    x, y = Tensor([1, 2, 3]), Tensor([4, 5, 6])
+    grid_x, grid_y = x.meshgrid(y)
+    print("\\n".join([repr(x.numpy()) for x in (grid_x, grid_y)]))
```

Comment:
just print in two lines, that's what the doc reader cares about and str join repr numpy is confusing